package.json

```diff
@@ -18,8 +18,8 @@
         "dev:service:swap_http": "nodemon -r dotenv/config src/runners/http_swap_service_runner.ts | pino-pretty",
         "dev:service:order_watcher": "nodemon -r dotenv/config src/runners/order_watcher_service_runner.ts | pino-pretty",
         "watch": "tsc -w",
-        "prettier": "prettier --write '**/*.{ts,tsx,json,md}' --config .prettierrc",
-        "prettier:ci": "prettier --list-different '**/*.{ts,tsx,json,md}' --config .prettierrc",
+        "prettier": "prettier --write 'src/**/*.{ts,tsx,json,md}' --config .prettierrc",
```

Comment:
consider .prettierignore

Reply:
This is the output I get without limiting it to `src`:

```
$ yarn prettier:ci
yarn run v1.19.1
$ prettier --list-different './**/*.{ts,tsx,json,md}' --config .prettierrc
[error] Unable to expand glob patterns: ./**/*.{ts,tsx,json,md} !**/node_modules/** !./node_modules/** !**/.{git,svn,hg}/** !./.{git,svn,hg}/**
[error] EACCES: permission denied, scandir '/home/gene/dev/0x-api/postgres'
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

That `postgres` folder was created by a docker container, and has root-only permissions:
```
$ ls -ld postgres
drwx------ 19 999 root 4096 Apr  8 16:04 postgres
```

I tried using a .pretterignore file, with values of `postgres`, `/postgres`, and `/postgres/`, and none changed this behavior.

Reply:
Hrm, I don't like that we are not linting `package.json` anymore.  It does seem like this is a linux-specific issue where the `scandir` still occurs regardless of `.prettierignore`

Could you possibly just `chown` the `postgres` dir after it's created?  Or is it recreated every time you `docker-compose up`?

Reply:
It's recreated every time I `docker-compose up`.

Does this address your concerns? https://github.com/0xProject/0x-api/pull/162/commits/f325f428a8dd863527d0870ee0e1a08438e2a0c6

Reply:
As long as you tested this as it works as expected, this commit LGTM 👍